admin-api-server/src/analytics/service/analytics.service.ts

```diff
@@ -181,7 +189,10 @@ FROM (
     usr.address AS "to",
     nft.id AS token_id,
     nft.price,
-    1 AS amount
+    1 AS edition_size,
+    payment.currency,
+    SUM(nft.price) OVER (PARTITION BY payment.nft_order_id) AS transaction_value,
```

Comment:
This will break if someone adds a where clause that reduces the rows under inspection, resulting in unexpected nft_order sum prices. It's probably the most efficient in terms of execution speed but I think it's a bit too brittle. A better way is to replace this line with:

```(
SELECT 
  SUM(nft.price) 
FROM nft 
JOIN mtm_nft_order_nft AS mtm 
   ON mtm.nft_order_id = nft_order.id
)
```

(and only if/when this becomes horribly slow a solution would be to go with your current approach or perhaps some other solution).

Reply:
Done, thank you.